distributions/distribution-resources/src/main/resources/etc/org.ops4j.pax.url.mvn.cfg

```diff
@@ -56,47 +56,28 @@ org.ops4j.pax.url.mvn.useFallbackRepositories=false
 
-#
-# Comma separated list of repositories scanned when resolving an artifact.
-# Those repositories will be checked before iterating through the
-#    below list of repositories and even before the local repository
-# A repository url can be appended with zero or more of the following flags:
-#    @snapshots  : the repository contains snaphots
-#    @noreleases : the repository does not contain any released artifacts
-#
-# The following property value will add the system folder as a repo.
-#
-org.ops4j.pax.url.mvn.defaultRepositories=\
-    file:${karaf.home}/${karaf.default.repository}@id=system.repository@snapshots
-#    file:${karaf.data}/kar@id=kar.repository@multi@snapshots, \
-#    file:${karaf.base}/${karaf.default.repository}@id=child.system.repository@snapshots
-
-# Use the default local repo (e.g.~/.m2/repository) as a "remote" repo
-#org.ops4j.pax.url.mvn.defaultLocalRepoAsRemote=false
 
 #
 # Comma separated list of repositories scanned when resolving an artifact.
-# The default list includes the following repositories:
-#    http://repo1.maven.org/maven2@id=central
-#    http://repository.springsource.com/maven/bundles/release@id=spring.ebr
-#    http://repository.springsource.com/maven/bundles/external@id=spring.ebr.external
-#    http://zodiac.springsource.com/maven/bundles/release@id=gemini
-#    http://repository.apache.org/content/groups/snapshots-group@id=apache@snapshots@noreleases
-#    https://oss.sonatype.org/content/repositories/snapshots@id=sonatype.snapshots.deploy@snapshots@noreleases
-#    https://oss.sonatype.org/content/repositories/ops4j-snapshots@id=ops4j.sonatype.snapshots.deploy@snapshots@noreleases
-# To add repositories to the default ones, prepend '+' to the list of repositories
-# to add.
+# 
 # A repository url can be appended with zero or more of the following flags:
 #    @snapshots  : the repository contains snapshots
 #    @noreleases : the repository does not contain any released artifacts
 #    @id=repository.id : the id for the repository, just like in the settings.xml this is optional but recommended
 #
-org.ops4j.pax.url.mvn.repositories=
-#    http://repo1.maven.org/maven2@id=central, \
-#    http://repository.springsource.com/maven/bundles/release@id=spring.ebr.release, \
-#    http://repository.springsource.com/maven/bundles/external@id=spring.ebr.external, \
-#    http://zodiac.springsource.com/maven/bundles/release@id=gemini, \
-#    http://repository.apache.org/content/groups/snapshots-group@id=apache@snapshots@noreleases, \
-#    https://oss.sonatype.org/content/repositories/snapshots@id=sonatype.snapshots.deploy@snapshots@noreleases, \
-#    https://oss.sonatype.org/content/repositories/ops4j-snapshots@id=ops4j.sonatype.snapshots.deploy@snapshots@noreleases, \
-#    http://repository.springsource.com/maven/bundles/external@id=spring-ebr-repository@snapshots@noreleases
+# This contains the openhab, smart home and default repositories.
+#
+org.ops4j.pax.url.mvn.repositories= \
```

Comment:
What would be nice through is a possibility to update the content on an offline distro as well. I was actually hoping that we could have a addons.kar file somewhere in the distro, from where features are resolved. Users would have been able to download a newer version and replace it locally. Even for the online version, we could have the same kar made available at a single place. But somehow @maggu2810 struggled getting it working this way. I don't remember the details, but maybe kars are not meant to be used that way?